rust/psibase_macros/psibase-macros-lib/src/tests/basics.rs

```diff
@@ -48,4 +48,25 @@ mod tests {
 
         let _after = service_macro_impl(attr, before);
     }
+
+    #[test]
+    // `service` macro should expand without requiring a `use` to provide its deps
```

Comment:
It always expanded successfully. The generated code just didn't compile.

Reply:
ok, comment updated for accuracy, and this test is part of the suite that runs as part of CI/CD, so we know that it expands *and compiles*

Reply:
The original comment accurately described the test. It's just not a very useful test.